### PR TITLE
Add project dataset information

### LINKED ISSUE #22: List datasets in Repo
List datasets for the ecosystems that we have found and have interest in assessing.

### PR DESCRIPTION
**Pull request by nyghtowl**
Added a link to the dataset list and information on where to go to be part of the research group 

Fixes #22, #17 

Note for now asking the broader community to submit any datasets additions through an issue request. A more formal form will be looked at later.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,31 @@ We are focused on studying the following ecosystems.
 - Node
 - Python
 
+## Project Datasets 
+We are collecting a list of datasets that would be useful for this project based on our ecosystem focus. The link below provides the latest list which is an ongoing work in progress. 
+
+[OCEAN Open Soruce Ecosystem Data Map](https://docs.google.com/spreadsheets/d/e/2PACX-1vROZwJ7Bzzp58XHnS1LlfeJ5aLhKCugp9IzgaX5jCtZ9vki42Xrfy55HxJKAsrrDx1zZss4EsvbiQMX/pubhtml?gid=0&single=true)
```
Comment (amcasari) [inline README.md:22]: Correct spelling + make bold: **OCEAN Open Source Ecosystem Data Map**

File: README.md [hunk 2]
```diff
@@ -16,6 +16,31 @@ We are focused on studying the following ecosystems.
...
+
+If you know of a dataset that should be on the list or have updates to recommend, for now submit an Issue with as much of the following information that you have:
```
Comment (amcasari) [inline README.md:24]: Created an issue template with this information. PR #29 ready for your review.

File: README.md [hunk 3]
```diff
@@ -16,6 +16,31 @@ We are focused on studying the following ecosystems.
...
+- Size (GB/TB/?) - (if you record a different size, note it next to the number)
+- Accessible (How can we access it either API or scrape with permission or no access option or something else)
+- Start Date (When it started being collected - at a minimum what year)
+- End Date (When it stopped being collected or note today if its kept current - at a minimum what year)
+- Update Frequence (How often its updated - daily, monthly, etc)
+- Reference Links (especially dataset schema and other info that is useful)
+- Other Info
+
+
+## OCEAN External Faculty Program
```
Comment (amcasari) [inline README.md:40]: Recommend waiting on including this information until we have public details to post for how people can apply through UVM.

**Review by amcasari — COMMENTED** [review]
LGTM. A few comments on format and removing one section until we have more information.

File: README.md (hunk 3 above)
Comment (nyghtowl) [reply README.md:40]: Revised to share that the information is coming

**Pull request merged**